### ISSUE TITLE
Spermatogonial arrest

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9955]
**Preferred term label:**
Spermatogonial arrest

**Synonyms**

**Definition (free text, please give PubMed ID)**
Testicular histology shows seminiferous tubules with Sertoli cells and spermatogonial cells but no further differentiated cells like spermatocytes or round spermatids. 

**Parent term (use hpo.jax.org/app)**
HP:0031038

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NA

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as 
HP:4000187
